app/controllers/facility_reservations_controller.rb

```diff
@@ -221,32 +206,27 @@ def update_prices
     end
   end
 
-  def new_or_in_process_orders(order_by_clause = "reservations.reserve_start_at")
-    current_facility.order_details.new_or_inprocess.reservations
-                    .includes(
-                      { order: :user },
-                      :order_status,
-                      :reservation,
-                      :assigned_user,
-                    )
-                    .where("reservations.id IS NOT NULL")
-                    .order(order_by_clause)
-  end
-
   def problem_order_details
     current_facility
       .problem_reservation_order_details
-      .joins(:reservation)
+      .includes(:reservation)
       .order("reservations.reserve_start_at DESC")
```

Comment:
I think if we were doing a `where(reservations: {})` we would need the `references`. But we're just doing this to avoid an N+1 here, so I don't think it's necessary. It works, so I'm guessing it's not.